src/utils/addresses.ts

```diff
@@ -11,6 +11,10 @@ export function isAddress(value: any): string | false {
   }
 }
 
+export function isSameAddress(a?: string, b?: string) {
```

Comment:
duped here now https://github.com/Uniswap/interface/blob/main/src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx#L80

Reply:
i removed this in this PR already